.github/workflows/ci.yml

```diff
@@ -118,7 +118,19 @@ jobs:
         run: make vendor vendor.check
 
       - name: Check Diff
-        run: make check-diff
+        run: |
+          mkdir _output
+          make check-diff
+        env:
+          # check-diff depends on the generate Make target, and we would like
+          # to save a skipped resource list
+          SKIPPED_RESOURCES_CSV: ../_output/skipped_resources.csv
+
+      - name: Publish skipped resources CSV to Github
```

Comment:
I believe this better fits `publish-artifacts` job below rather than `check-diff`. What do you think @ulucinar ?

Reply:
I agree. Unfortunately, the `make build.all` does not currently call the `generate` target and I did not want to introduce an extra `make generate` call that would make the workflows a little bit longer. I've discussed it [here](https://github.com/upbound/provider-aws/pull/139#issue-1444066034).